core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowResource.scala

```diff
@@ -401,4 +409,29 @@ class WorkflowResource {
     }
   }
 
+  /**
+    * This method updates the description of a given workflow
+    *
+    * @return Response
+    */
+  @POST
+  @Path("/update/description/{wid}/{workflowDescription}")
+  @Consumes(Array(MediaType.APPLICATION_JSON))
+  @Produces(Array(MediaType.APPLICATION_JSON))
+  def updateWorkflowDescription(
+      @PathParam("wid") wid: UInteger,
+      @PathParam("workflowDescription") workflowDescription: String,
```

Comment:
Since it's a POST request, it's better to put the parameters in the request body, instead of in the path. The path parameter is more commonly used for GET requests. Check how other POST requests endpoints work in the backend and frontend.